go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/ledgerwatch/erigon
 
-go 1.21
+go 1.21.5
```

Comment:
Why bump golang version?

Reply:
probably  "go mod tidy" did it, I'll revert it

Reply:
Fixed